Sources/TokamakCore/Tokens/Color/Color.swift

```diff
@@ -130,6 +130,20 @@ public extension Color {
   }
 }
 
+public extension ShapeStyle where Self == Color {
```

Comment:
Could you include the new colors from the beta here (brown, cyan, indigo, mint, teal)?